web/beacon-app/cypress/e2e/ProjectDetailPage.feature

```diff
@@ -1,5 +1,71 @@
-describe('template spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
-  })
-})
\ No newline at end of file
+Feature: Project Detail Page
+
+I want to navigate a project detail page
+
+Scenario: Navigating a project detail page
+
+Given I'm logged into Beacon
+And I click Projects
+Then I navigate to the project page
+And I click on a project in the project table list
+Then I should see the project detail page for the project
+And I should see the project name at the top of the page
+#And I should see the project's details when I hover over the hint icon next to the project name
```

Comment:
I had trouble testing hovering over the hint icons in Cypress. I tried the suggestions in the docs and sometimes `trigger` or `invoke` would work, but then the same tests would fail again. I also tried using `cypress-real-events`. I'll create a separate story to try and figure that out, but didn't want to hold this test up any longer.